### ISSUE TITLE
Fluvio CMS | Collect list of connectors dynamically | `failed to copy` when generating docs for connector with CLI

### ISSUE THREAD

**Issue by EstebanBorai** [infinyon/fluvio-website#564]
When attempting to generate new docs for a connector using:

```bash
cargo run -- connector --service http --direction outbound
```

An error on copy is thrown:

```logs
2023-05-02T22:47:12.606912Z  INFO fluvio_cms::connectors::cli: Write the README file to embeds/connectors/outbound/http.md
Error: failed to copy `/var/folders/vx/lkxmsbr95dd4r0j1pwkg0skm0000gn/T/xshell-tmp-dir-0/README.md` to `/fluvio-website/embeds/connectors/outbound/http.md`: No such file or directory (os error 2)
```

**Comment by EstebanBorai** [comment]
Encountered while working on: https://github.com/infinyon/fluvio-website/pull/561

**Comment by morenol** [comment]
I think that we need to fix this https://github.com/infinyon/fluvio/issues/3197 (we need to include readme in the connector package).

On the other hand, the connectors used seem to be these: https://github.com/infinyon/fluvio-website/blob/4879ba86db80b968a5c37cb2d67f58aee6e0652e/scripts/fluvio-cms/src/connectors/mod.rs#L18

Maybe we need to use the version that we get when we do `fluvio hub connector list` instead

**Comment by tjtelan** [comment]
> I think that we need to fix this [infinyon/fluvio#3197](https://github.com/infinyon/fluvio/issues/3197) (we need to include readme in the connector package).
> 
> On the other hand, the connectors used seem to be these:
> 
> https://github.com/infinyon/fluvio-website/blob/4879ba86db80b968a5c37cb2d67f58aee6e0652e/scripts/fluvio-cms/src/connectors/mod.rs#L18
> 
> Maybe we need to use the version that we get when we do `fluvio hub connector list` instead

Yeah, that's a good approach, assuming that list has the most recent version for all the public connectors

**Comment by morenol** [comment]
Seems to be working now. Will create an issue for the automatic last connector version resolution
